src/api/movieService.ts

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios'
+import type { Movie } from '@/types/movie'
+
+// axios.defaults.baseURL = import.meta.env.VITE_API;
+axios.defaults.baseURL = 'http://localhost:3000';
```

Comment:
Was this omitted by mistake or do you see any issues using global variable?

Reply:
I was testing must have not cleanup